### PR TITLE
Jackieg/azuretheme cal

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
This PR removes the ContextMenu harsh border, updates the DatePicker/Calendar "today is selected" blue and gray selections, reduced the dropdown items size to 32px, and removed the focus border from the pivot - all for Azure Theme only. 

## Previous Behavior

Context Menu harsh border
image
image

Dropdown item height previously 36px
image

Pivots had a focus border, even when just pressed.
image

## New Behavior
image

image

with header item 
image

Dropdown item height updated to 32px
image

Pivot button:focus border removed
image

### CODE REVIEW

File: packages/azure-themes/src/azure/styles/DetailsList.styles.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const CheckStyles = (props: ICheckStyleProps): Partial<ICheckStyles> => {
         fontSize: 0,
         paddingTop: 1,
         paddingLeft: 1,
-        borderRadius: 2,
+        borderRadius: StyleConstants.borderRadius,
```
Comment (Jacqueline-ms) [inline packages/azure-themes/src/azure/styles/DetailsList.styles.ts:22]: Updated all borderRadius=2 to use constants

File: packages/react-examples/src/azure-themes/stories/components/CalendarInlineMultidayDayView.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```
Comment (Jacqueline-ms) [inline packages/react-examples/src/azure-themes/stories/components/CalendarInlineMultidayDayView.stories.tsx:1]: Added another calendar example from Fluent

File: packages/react-examples/src/azure-themes/stories/components/ProgressIndicator.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```
Comment (Jacqueline-ms) [inline packages/react-examples/src/azure-themes/stories/components/ProgressIndicator.stories.tsx:1]: Added progressindicator control, as it was using another semanticcolor I wanted to check on

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b9284238a8b79e3d0864ce26a8bb0e9ffb33a96a'>b9284238a8b79e3d0864ce26a8bb0e9ffb33a96a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-azure-themes-fde6e654-0910-4897-912e-842bf2024601.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-azure-themes-fde6e654-0910-4897-912e-842bf2024601.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26544&commitId=61baa11e7827262197e6bc23d5fda55d2e4ce678&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

File: packages/azure-themes/src/azure/Constants.ts [hunk 5]
```diff
@@ -6,10 +6,11 @@ import {
 } from '@fluentui/react/lib/Styling';
 
 export const borderNone = 'none';
-export const borderRadius = '3px';
+export const borderRadius = '2px';
```
Comment (Jacqueline-ms) [inline packages/azure-themes/src/azure/Constants.ts:9]: Updated global borderRadius from 3px to 2px per designers

File: packages/azure-themes/src/azure/styles/DetailsList.styles.ts [hunk 6]
```diff
@@ -29,8 +29,8 @@ export const CheckStyles = (props: ICheckStyleProps): Partial<ICheckStyles> => {
     ],
     check: [
       {
-        left: 2.5,
-        top: 1.5,
+        left: 1.8,
+        top: 0.5,
```
Comment (Jacqueline-ms) [inline packages/azure-themes/src/azure/styles/DetailsList.styles.ts:33]: Properly centered the DetailsList check

File: packages/azure-themes/src/azure/styles/Pivot.styles.ts [hunk 7]
```diff
@@ -99,6 +99,9 @@ export const PivotStyles = (props: IPivotStyleProps): Partial<IPivotStyles> => {
           ':active': {
             backgroundColor: semanticColors.bodyBackground,
           },
+          ':focus': {
+            border: 0,
+          },
```
Comment (Jacqueline-ms) [inline packages/azure-themes/src/azure/styles/Pivot.styles.ts:104]: Removed the focus border. When user clicks button, the border appeared.

**Review by robtaft-ms — APPROVED** [review]
This looks good thank you!

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Pull request merged**